src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -3,15 +3,19 @@
 
 using System;
 using System.IO;
+using System.Reflection;
 using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Configuration.UserSecrets.Internal;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Configuration extensions for adding user secrets configuration source.
+    /// </summary>
     public static class ConfigurationExtensions
     {
-        private const string Secrets_File_Name = "secrets.json";
-
+#if NET451 || NETSTANDARD1_5
```

Comment:
I think you missed my point. _This is not going to break existing apps._ 
With a nupkg that targets net451, netstandard1.3 and netstandard1.5, the only existing platform that would use the netstandard1.3 binary is UWP. We don't support UWP. Microsoft.Extensions.Configuration.UserSecrets 1.0.0 doesn't even install into a UWP project. 
.NET Core apps will get the netstandard1.5 binary, .NET Framework will get the net451 binary, both of which preserve existing API and functionality.